OptimizelySDKCore/OptimizelySDKCore/Optimizely.m

```diff
@@ -215,7 +215,7 @@ - (BOOL)isFeatureEnabled:(NSString *)featureKey userId:(NSString *)userId attrib
     
     OPTLYFeatureDecision *decision = [self.decisionService getVariationForFeature:featureFlag userId:userId attributes:attributes];
     
-    if (!decision) {
+    if (!decision || !decision.variation.featureEnabled) {
```

Comment:
Can we also null check the `variation` or does obj-c handle nil values gracefully?

Reply:
we are checking null variation in the GetVariationForFeature method. I guess we shouldn't check here.